UI/forms/source-toolbar/device-select-toolbar.ui

```diff
@@ -57,7 +57,7 @@
       <string notr="true"/>
      </property>
      <property name="sizeAdjustPolicy">
-      <enum>QComboBox::AdjustToMinimumContentsLength</enum>
+      <enum>QComboBox::AdjustToContents</enum>
```

Comment:
Jim just recently said that he would not change `QComboBox::AdjustToMinimumContentsLength` to `QComboBox::AdjustToContents` because the behavior is not the same.  See [this comment](https://github.com/obsproject/obs-studio/pull/3321#issuecomment-678815669) on #3321 for details.